src/inventory_ui.cpp

```diff
@@ -218,6 +218,11 @@ inventory_entry *inventory_column::find_by_invlet( int invlet ) const
     return nullptr;
 }
 
+void group_entry::set_number_of_children( const int n )
```

Comment:
```suggestion
void group_entry::set_number_of_children( const int &n )
```

Reply:
It's fine to (maybe even cheaper to) copy ints, the function definition should just lose the const.

Reply:
Isn't reference cheaper?

Reply:
Wasn't thinking much about it, other than needs to be enclosed, but yes it's probably way overkill.

Reply:
Not for an int.
```cpp
#include <cstdio>
#include <string>

int main()
{
    class intref
    {
        int &ref;
        intref(int a) : ref(a) {}
    };
    class stringref
    {
        std::string &ref;
        stringref(std::string a) : ref(a) {}
    };
    printf("sizeof int %ld\n", sizeof(int));
    printf("sizeof int & %ld\n", sizeof(intref));
    printf("sizeof int * %ld\n", sizeof(int *));

    printf("sizeof std::sting %ld\n", sizeof(std::string));
    printf("sizeof std::string & %ld\n", sizeof(stringref));
    printf("sizeof std::string * %ld\n", sizeof(std::string *));
    return 0;
}
>>>>>
sizeof int 4
sizeof int & 8
sizeof int * 8
sizeof std::sting 32
sizeof std::string & 8
sizeof std::string * 8
```

Reply:
Hah, on x86 would be a wash, though.

Reply:
It's such a micro optimization it doesn't matter, but copying is fine for ints and many of the basic types.

Reply:
A copy can be passed directly via a register, while a reference has to be accessed via the memory, so I'd say a copy of a scalar is always cheaper.